doc/source/install/install.md

```diff
@@ -351,6 +366,278 @@ sudo -i python $PWD/iaas_launch.py -f $PWD/conf/pxe_cluster/hosts.yaml -b
 This will boot host machines (controller/compute nodes), select
 NIC Controller (PXE client enabled) to use network booting.
 
+This will provision ubuntu OS on host machines.
+
+Your OS provisioning will start and will get completed in about 20
+minutes.  The time will vary depending on your network speed and
+server boot times.
+
+#### Step 7
```

Comment:
A lot of these instructions are duplicated from the Ubuntu section.  Can the CentOS specific steps be reduced to the items that are specific to CentOS?

Please be clear that this is using an Ubuntu 16.04 server to install CentOS 7.

Reply:
Updated the install.md as per the suggestions from Randy.